new/new.go

```diff
@@ -44,13 +44,22 @@ func New(templName string, targets []string) error {
 		templSchemaPath = filepath.Join(dotcfg.SchemasDirName, templObj.FilePath)
 	}
 
-	args := append([]string{"init", "-s", templSchemaPath}, targets...)
-	cmd := exec.Command("schema", args...)
-	if err := cmdrunner.PipeTo(cmd, os.Stdout, os.Stderr); err != nil {
-		return err
+	if !keepVals {
+		args := append([]string{"init", "-s", templSchemaPath}, targets...)
+		cmd := exec.Command("schema", args...)
+		if err := cmdrunner.PipeTo(cmd, os.Stdout, os.Stderr); err != nil {
+			return err
+		}
```

Comment:
need to `return` at the end of this `if` statement, or the rest of the code will erroneously execute

Reply:
disregard